discovery-provider/es-indexer/src/indexers/TrackIndexer.ts

```diff
@@ -189,7 +189,7 @@ export class TrackIndexer extends BaseIndexer<TrackDoc> {
     row.tags = row.tags
     row.repost_count = row.reposted_by.length
     row.favorite_count = row.saved_by.length
-    row.duration = Math.ceil(
+    row.length = Math.ceil(
```

Comment:
doesn't this need to be duration to match the search response?

Reply:
`length` is the name on the actual sql table... but I guess `duration` is in the API response.
Maybe I should just write it to both fields?

Reply:
maybe we should translate length to mean duration in `transform_tracks` or `populate_track_or_playlist_metadata_es`? i'm ok with either

Reply:
I updated `es-indexer` to populate both `length` and `duration` for now.